web/README.md

```diff
@@ -5,8 +5,8 @@ containing the settings to build the website. The website is generated with the
 command `./pandas_web.py pandas`. See `./pandas_web.py --help` and the header of
 the script for more information and options.
 
-After building the website, to navigate it, it is needed to access the web using
-an http server (a not open the local files with the browser, since the links and
+After building the website, the web must be accessed using
```

Comment:
Maybe just:

`After building the website, a local http sever is needed to access the built website (it is ...`

Reply:
Good suggestion, updated to reflect refinement.